sdk/eventhubs/azure-messaging-eventhubs-stress/stress-test-resources.bicep

```diff
@@ -1,41 +1,35 @@
 @description('The base resource name.')
-param baseResourceName string = resourceGroup().name
+param baseName string = resourceGroup().name
 
 @description('The location of the resources. By default, this is the same as the resource group.')
 param location string = resourceGroup().location
 
-var subBaseName = '${substring(baseResourceName, 0, min(length(baseResourceName), 22))}-${substring(guid(baseResourceName), 0, 5)}'
-var eventHubsNamespaceName = '${subBaseName}-namespace'
-var eventHubName = 'test-event-hub'
 var eventHubsAuthRulesName = 'stress-test-auth-rule'
-var secondEventHubName = 'test-event-hub-2'
-
-// storage account must be between 3 and 24 characters in length and use numbers and lower-case letters only
-var storageAccountName = replace('${subBaseName}ac', '-', '')
 var storageContainerName = 'test-blob-container'
 
 resource eventHubsNamespace 'Microsoft.EventHub/namespaces@2021-11-01' = {
-  name: eventHubsNamespaceName
+  name: baseName
   location: location
   sku: {
-    name: 'Standard'
-    tier: 'Standard'
+    name: 'Premium'
+    tier: 'Premium'
+    capacity: 2
```

Comment:
Just making sure, is it intentional to use _Premium_ for stress test runs?

Reply:
absolutely! otherwise results are unreliable (standard perf is not guaranteed) and we test much higher throughput that standard allows.

Reply:
Make sense, thanks for confirming